parsl/providers/slurm/slurm.py

```diff
@@ -133,6 +139,11 @@ def __init__(self,
             self.scheduler_options += "#SBATCH --partition={}\n".format(partition)
         if account:
             self.scheduler_options += "#SBATCH --account={}\n".format(account)
+        if queue:
```

Comment:
Maybe this parameter be called `qos` so to match the SLURM vocabulary? (eg. compare to sites that use `partition` as their queue abstraction, not `qos`: https://help.jasmin.ac.uk/article/4890-how-to-submit-a-job-to-slurm)